cp3pt0-deployment/setup_tenant.sh

```diff
@@ -142,6 +143,7 @@ function print_usage() {
 }
 
 function pre_req() {
+    title "Start to validate necessary functional prerequisite... "
```

Comment:
Could we update the title here that `pre_req` is used to validate the parameters passed into script.

functional prerequisite is something like certificate management functionality.